### PR TITLE
Rendering test for `box-shadow` with `calc()`

### PR DESCRIPTION
**Pull request by graouts**
This was added in [WebKit revision 282768](https://commits.webkit.org/241903@main) for [bug 230347](https://bugs.webkit.org/show_bug.cgi?id=230347).

### CODE REVIEW

File: css/css-backgrounds/box-shadow-calc.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<link rel="author" title="Antoine Quint" href="mailto:graouts@apple.com" />
+<link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#the-box-shadow" />
+<link rel="match" href="box-shadow-calc-ref.html" />
+<meta name="assert" content="Test checks whether box-shadow supports calc() values.">
+<style type="text/css">
+div {
+  width: 100px;
+  height: 100px;
+  background-color: blue;
+  box-shadow: calc(1em + 10px) calc(1em + 10px) calc(1em + 10px) black;
```
Comment (smfr) [inline css/css-backgrounds/box-shadow-calc.html:11]: You should probably explicitly specify a font and size so that 1em is the same across all browsers.

Comment (dbaron) [reply css/css-backgrounds/box-shadow-calc.html:11]: It used to be documented that it was safe to assume a `16px` default font size in CSS/WPT reftests, though I can't find that documentation anymore...

**Review by dbaron — APPROVED** [review]
Looks good to me, although in the future it's sometimes good to use different numbers when writing this sort of test (i.e., three different resulting values rather than 26px 26px 26px) since that also tests for the case where the values get crossed.

**Comment by graouts** [comment]
> Looks good to me, although in the future it's sometimes good to use different numbers when writing this sort of test (i.e., three different resulting values rather than 26px 26px 26px) since that also tests for the case where the values get crossed.

I'll fix this so this test is as good as it can be.

File: css/css-backgrounds/box-shadow-calc.html (hunk 1 above)
Comment (graouts) [reply css/css-backgrounds/box-shadow-calc.html:11]: 👍

Comment (gsnedders) [reply css/css-backgrounds/box-shadow-calc.html:11]: https://github.com/web-platform-tests/wpt/blob/7b0ebaccc62b566a1965396e5be7bb2bc06f841f/docs/writing-tests/assumptions.md seems to be it today

**Pull request merged**